src/DDTrace/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -436,6 +435,54 @@ function ($This, $scope, $args, $loginSuccess) use ($integration) {
             }
         );
 
+        // Used by Laravel < 5.0
+        \DDTrace\hook_method(
```

Comment:
I might be off here, but isn't the new stuff supposed to use `\DDTrace\install_hook`?